react/CheckOrganizationPermission.tsx

```diff
@@ -0,0 +1,78 @@
+import React from 'react'
+import type { ElementType } from 'react'
+import { ExtensionPoint } from 'vtex.render-runtime'
+import { useQuery } from 'react-apollo'
+
+import getOrganizationPermissions from './queries/getOrganizationPermissions.gql'
```

Comment:
This query is not included in the PR, did you forget to commit it? It's not present on the codebase too

Reply:
resolved